src/Resources/contao/classes/SurveyQuestionMultiplechoice.php

```diff
@@ -255,27 +255,27 @@ protected function calculateAnsweredSkipped(& $objResult): void
         $this->arrStatistics['skipped'] = 0;
 
         while ($objResult->next()) {
-            $id = \strlen($objResult->pin) ? $objResult->pin : $objResult->uid;
+            $id = !empty($objResult->pin) ? $objResult->pin : $objResult->uid;
             $this->arrStatistics['participants'][$id][] = $objResult->row();
             $this->arrStatistics['answers'][] = $objResult->result;
 
-            if (\strlen($objResult->result)) {
+            if (!empty($objResult->result)) {
                 $arrAnswer = StringUtil::deserialize($objResult->result, true);
                 $found = false;
 
                 if (\is_array($arrAnswer['value'])) {
                     foreach ($arrAnswer['value'] as $answervalue) {
-                        if (\strlen($answervalue)) {
+                        if (!empty($answervalue)) {
                             $found = true;
                         }
                     }
                 } else {
-                    if (\strlen($arrAnswer['value'])) {
+                    if (!empty($arrAnswer['value'])) {
                         $found = true;
                     }
                 }
 
-                if (\strlen($arrAnswer['other'])) {
+                if (!empty($arrAnswer['other'])) {
```

Comment:
Using `strlen` can lead to the following error:

```
TypeError:
strlen(): Argument #1 ($str) must be of type string, null given

  at vendor\pdir\contao-survey\src\Resources\contao\classes\SurveyQuestionMultiplechoice.php:278
```

This PR fixes that by using `!empty()` instead of `strlen()`.